content/docs/2.10/authentication-providers/azure-ad-workload-identity.md

```diff
@@ -23,3 +23,32 @@ following flags -
 3. `--set podIdentity.azureWorkload.tenantId={azure-ad-tenant-id}`
 
 You can override the identity that was assigned to KEDA during installation, by specifying an `identityId` parameter under the `podIdentity` field. This allows end-users to use different identities to access various resources which is more secure than using a single identity that has access to multiple resources.
+
+## Considerations about Federations and Overrides
+
+The concept of "overrides" can be somewhat confusing in certain scenarios, as it may not always be clear which service account needs to be federated with a specific Azure Identity to ensure proper functionality. Let's clarify this with two examples:
```

Comment:
```suggestion
The concept of "overrides" can be somewhat confusing in certain scenarios, as it may not always be clear which service account needs to be federated with a specific Azure identity to ensure proper functionality.

Let's clarify this with two examples:
```